### PR TITLE
Update OneSmallStep v0.9.6 > v0.9.7

### PR DESCRIPTION
**Pull request by BenTalagan**
Changelog : 

  - [Feature] Added repitch (+revel) mode (thanks @smandrap !)
  - [Fix] Added missing "Change Edit Mode" actions
  - [Doc] Added help button that redirects to current documentation
  - [Rework] Re arranged settings panel
  - [Rework] Reworked some icons and colors
  - [Rework] Started to use MIDI Utils API by sockmonkey72 instead of default MIDI API

### CODE REVIEW

File: MIDI Editor/talagan_OneSmallStep.lua [hunk 1]
```diff
@@ -9,6 +9,11 @@
   [main=main,midi_editor] talagan_OneSmallStep/actions/talagan_OneSmallStep Change input mode.lua             > talagan_OneSmallStep/actions/talagan_OneSmallStep Change input mode - KeyboardPress.lua
```
Comment (cfillion) [inline MIDI Editor/talagan_OneSmallStep.lua:9]: MIDIUtils.lua is not listed in the included files:

```
  [nomain] talagan_OneSmallStep/lib/*.lua
```

Comment (BenTalagan) [reply MIDI Editor/talagan_OneSmallStep.lua:9]: Yeah, it's in `talagan_OneSmallStep/classes/lib` and I stupidly thought that

`[nomain] talagan_OneSmallStep/classes/*.lua`

was recursive. Happily you were vigilant !! Correcting this right away.

Comment (BenTalagan) [reply MIDI Editor/talagan_OneSmallStep.lua:9]: Done. Thanks for your watchfulness !

File: MIDI Editor/talagan_OneSmallStep.lua [hunk 2]
```diff
@@ -34,40 +39,28 @@
...
   [main=main,midi_editor] talagan_OneSmallStep/actions/talagan_OneSmallStep Edit Action.lua                   > talagan_OneSmallStep/actions/talagan_OneSmallStep Edit Action - NavigateBack.lua
+  [main=main,midi_editor] talagan_OneSmallStep/actions/talagan_OneSmallStep Edit Action.lua                   > talagan_OneSmallStep/actions/talagan_OneSmallStep Edit Action - Repitch.lua
+  [main=main,midi_editor] talagan_OneSmallStep/actions/talagan_OneSmallStep Edit Action.lua                   > talagan_OneSmallStep/actions/talagan_OneSmallStep Edit Action - RepitchBack.lua
   [main=main,midi_editor] talagan_OneSmallStep/actions/talagan_OneSmallStep Increase note len.lua
   [main=main,midi_editor] talagan_OneSmallStep/actions/talagan_OneSmallStep Decrease note len.lua
   [main=main,midi_editor] talagan_OneSmallStep/actions/talagan_OneSmallStep Cleanup helper JSFXs.lua
   [main=main,midi_editor] talagan_OneSmallStep/actions/talagan_OneSmallStep Set or remove playback marker.lua
   [main=main,midi_editor] talagan_OneSmallStep/actions/talagan_OneSmallStep Playback.lua
   [nomain] talagan_OneSmallStep/classes/*.lua
+  [nomain] talagan_OneSmallStep/classes/lib/*.lua
```
Comment (cfillion) [inline MIDI Editor/talagan_OneSmallStep.lua]: There's a syntax for including a directory recursively:
```suggestion
  [nomain] talagan_OneSmallStep/classes/**/*.lua
```

(Will merge tomorrow, it's very late now.)

File: MIDI Editor/talagan_OneSmallStep.lua (hunk 1 above)
Comment (BenTalagan) [reply MIDI Editor/talagan_OneSmallStep.lua:9]: Thanks for the tip, I've accepted your suggestion. Bonne nuit ! :)

**Pull request merged**